src/IntervalArithmetic.jl

```diff
@@ -64,7 +64,8 @@ export
     cancelminus, cancelplus, isunbounded,
     .., @I_str, ±,
     pow, extended_div,
-    setformat, @format
+    setformat, @format , vector_sum, vector_dot,
+    vector_sumAbs, vector_sumSquare
```

Comment:
I think that would be more Julian to extend base function with the following methods:

```julia
sum(v::Vector, r::RoundingMode)
sum(f::Function, v::Vector, r::RoundingMode)
dot(v::Vector, r::RoundingMode)
```
That way `vector_sumAbs` and `vector_sumSquare` are given by respectively `sum(abs, v, r)` and `sum(abs2, v, r)`.

Reply:
Strictly speaking that is type piracy, but I agree that makes more sense.